tests/chainer_tests/functions_tests/noise_tests/test_dropout.py

```diff
@@ -10,15 +10,21 @@
 
 
 # TODO(Kenta OONO): This test fixture check types only. Add more detailed test.
+@testing.parameterize(
+    {'dtype': numpy.float16},
+    {'dtype': numpy.float32},
+    {'dtype': numpy.float64},
+)
```

Comment:
I fixed it. And I found that the current implementation doesn't support zero-dim array. I'll make another pr for this problem.